src/Main.elm

```diff
@@ -1,10 +1,209 @@
 module Main exposing (main)
 
-import Data.Audience
-import Data.AudienceFolder
-import Html exposing (Html)
+import Browser
+import Data.Audience as A
+import Data.AudienceFolder as F
+import Dict
+import Html
+import Html.Attributes as Hat
+import Html.Events as Hev
+import Json.Decode as Jd
+import Parser as P
```

Comment:
I agree it's confusing, and have changed it to DataParser. I like having 'Parser' in the name because I like the way this module catches as many errors in the data as possible up front, i.e. I like the 'Parse not validate' school of thought.